### PR TITLE
[Enhancement][BugFix] Jboss Fuse 6 build fixes & enhancements

### PR DESCRIPTION
**Pull request by redhatrises**
- Update Makefile to be able to build with regular OVAL and OVAL5.11
- Fix deprecated OVAL tags
- Fix cpe_generate.py to use
- Part of #1046

### CODE REVIEW

File: JBoss/Fuse/6/Makefile [hunk 1]
```diff
@@ -101,7 +123,19 @@ validate-xml:
...
 validate: validate-xml
-	cd $(OUT); ../$(SHARED)/$(UTILS)/verify-references.py --rules-with-invalid-checks --ovaldefs-unused ssg-$(PROD)-xccdf.xml
+ifeq ($(OVAL_5_11), 0)
+	cd $(OUT); ../$(SHARED)/$(UTILS)/verify-references.py --rules-with-invalid-checks --ovaldefs-unused $(ID)-$(PROD)-xccdf.xml
+else
+	# If we are building against oscap version not supporting OVAL-5.11 language version yet,
+	# don't call verify-references.py with "--rules-with-invalid-checks" argument, since the
+	# OVAL checks using the 5.11 OVAL version will not be included in that case
+	@echo -e "\nWarning:\n"
+	@echo -e "\tRHEL/7 content build using oscap not supporting OVAL-5.11 language version detected!"
```
Comment (iankko) [inline JBoss/Fuse/6/Makefile]: Warning is OK. But it should be "\tJBoss/Fuse/6" instead of ```RHEL/7```.

File: JBoss/Fuse/6/Makefile [hunk 2]
```diff
@@ -101,7 +123,19 @@ validate-xml:
...
+	@echo -e "\tSince the OVAL-5.11 RHEL/7 OVAL checks are missing, will skip test for referenced,"
```
Comment (iankko) [inline JBoss/Fuse/6/Makefile]: Here to s#RHEL/7#JBoss/Fuse/6#

File: JBoss/Fuse/6/Makefile [hunk 3]
```diff
@@ -101,7 +123,19 @@ validate-xml:
...
+	@echo -e "\tbut undefined OVAL definitions during content validation. Consider building RHEL/7"
```
Comment (iankko) [inline JBoss/Fuse/6/Makefile]: And yet once.

**Comment by iankko** [comment]
Also, issuing ```make content``` with oscap supporting OVAL-5.11 (openscap-1.2.5-3.el7.x86_64) returns error due to missing ```input/oval/oval_5.11``` directory:
```
# make content
...
# Search ../../../shared/oval/oval_5.11 and input/oval/oval_5.11 directories to find all product specific
# OVAL-5.11 checks, which are regular files (not symlinks). Merge the final list into build/fuse6_oval
# directory
find ../../../shared/oval/oval_5.11 input/oval/oval_5.11 -maxdepth 1 -type f -name '*.xml' -exec cp {} build/fuse6_oval ';'
find: ‘input/oval/oval_5.11’: No such file or directory
make: *** [checks] Error 1
```

**Comment by iankko** [comment]
Besides the above issues, this PR looks good to me.

**Comment by redhatrises** [comment]
@iankko forgot to fix those before sending in the PR. They have been fixed now.

**Comment by iankko** [comment]
This version looks good to me. Thank you for the update && changes itself!. ACK.

**Pull request merged**